### PR TITLE
Restore -v in set deploymenthook and status 

### LINKED ISSUE #20097: Restore -v in set deployment-hook and status commands
As part of the 1.11 rebase (https://github.com/openshift/origin/pull/20033) we've dropped `-v` support in `oc set deployment-hook` and `oc status` commands we need to restore them back.

### PR DESCRIPTION
**Pull request by soltysh**
Fixes #20097

/assign @deads2k @juanvallejo

### CODE REVIEW

File: pkg/oc/cli/status/status.go [hunk 1]
```diff
@@ -74,31 +74,29 @@ func NewStatusOptions(streams genericclioptions.IOStreams) *StatusOptions {
 // NewCmdStatus implements the OpenShift cli status command.
 // baseCLIName is the path from root cmd to the parent of this cmd.
 func NewCmdStatus(name, baseCLIName, fullName string, f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
-	opts := NewStatusOptions(streams)
-
+	o := NewStatusOptions(streams)
 	cmd := &cobra.Command{
 		Use:     fmt.Sprintf("%s [-o dot | -s ]", StatusRecommendedName),
```
Comment (xingxingxia) [inline pkg/oc/cli/status/status.go]: The display of the short flag `-s` in the usage is confusing:
```
$ oc status -s
Error: flag needs an argument: 's' in -s
$ oc status -h
Usage:
  oc status [-o dot | -s ] [flags]
```
If `-s` stands for the global flag `--server`, it is unnecessary to be displayed under `oc status -h`
```
$ oc options
  -s, --server='': The address and port of the Kubernetes API server
```

**Comment by xingxingxia** [comment]
A. The following `-v` occurrence should be replaced with `--suggest`:
```
$ 3.11/oc status
...
5 infos identified, use 'oc status -v' to see details.
```
B. Just since 3.11, `-v` stands for global `--v`. However, this PR tries to restore `-v` as non-global flag in oc sub-commands. This is a problem.
```
$ 3.11/oc options
...
  -v, --v=0: log level for V logs
```

File: pkg/oc/cli/status/status.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ var (
 	  %[1]s -o dot | dot -T svg -o project.svg
 
 	  # See an overview of the current project including details for any identified issues.
-	  %[1]s -v`)
+	  %[1]s -s`)
```
Comment (xingxingxia) [inline pkg/oc/cli/status/status.go]: Same as https://github.com/openshift/origin/pull/20412/files#r205704850

File: pkg/oc/cli/status/status.go (hunk 1 above)
Comment (soltysh) [reply pkg/oc/cli/status/status.go]: There's not shorthand `-s` thanks for catching that up, should be `--suggest`.

**Comment by soltysh** [comment]
I fixed the stuff @xingxingxia pointed out, re-applying the label back.

**Pull request merged**